src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+
+function expandTree(treeIn) {
+  for (const item of treeIn.querySelectorAll('li')) {
+    if (item.querySelector('ul')) {
+      expandTree(item.querySelector('ul'));
+
+      const newSpan = document.createElement('span');
+
+      newSpan.textContent = item.firstChild.textContent.trim();
+
+      item.firstChild.remove();
```

Comment:
The line `item.firstChild.remove();` assumes that the first child of the list item is removable. If the first child is not a text node or if it doesn't exist, this could cause an error. Ensure that `item.firstChild` is not null before calling `remove()`.